data/governance/proposal_fund_store.go

```diff
@@ -120,6 +125,19 @@ func (pf *ProposalFundStore) AddFunds(proposalId ProposalID, fundingAddress keys
 	return pf.set(key, *amt.Plus(amount))
 }
 
+func (pf *ProposalFundStore) DeductFunds(proposalId ProposalID, fundingAddress keys.Address, amount *balance.Amount) error {
+	key := storage.StoreKey(string(proposalId) + storage.DB_PREFIX + fundingAddress.String())
```

Comment:
Do we need to delete the record if the amount drops to 0? Confirm with Tanmay?

Reply:
sure

Reply:
I think we decide to keep it.